app/src/main/res/layout/activity_main.xml

```diff
@@ -44,4 +54,16 @@
         app:backgroundTint="@android:color/holo_blue_dark"
         app:srcCompat="@android:drawable/stat_sys_data_bluetooth" />
 
+    <android.support.design.widget.FloatingActionButton
+        android:id="@+id/fab2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom|end"
+        android:layout_toLeftOf="@+id/fab"
+        android:clickable="true"
+        android:layout_marginBottom="80dp"
```

Comment:
In the other floating action button the following code is used to align it:
`android:layout_margin="@dimen/fab_margin"`
Can you make constants in dimens.xml from the marginBottom and marginRight?